apis/Google.Cloud.Firestore/Google.Cloud.Firestore.IntegrationTests/FirestoreFixture.cs

```diff
@@ -75,10 +87,21 @@ public FirestoreFixture() : base(ProjectEnvironmentVariable)
         {
             RunningOnEmulator = !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("FIRESTORE_EMULATOR_HOST"));
             AdminClient = FirestoreAdminClient.Create();
-            // Currently, only the default database is supported... so we create all our collections with a randomly-generated prefix.
-            // When multiple databases are supported, we'll create a new one per test run.
-            CollectionPrefix = IdGenerator.FromGuid(prefix: "test-");
-            FirestoreDb = new FirestoreDbBuilder { ProjectId = ProjectId, EmulatorDetection = EmulatorDetection.EmulatorOrProduction }.Build();
+
+            // On the emulator, we can't easily create a new database - so we just use the default.
+            // We expect each test run time be in a separate emulator session.
+            string databaseId = RunningOnEmulator ? null : IdGenerator.FromDateTime(prefix: "test-");
+            if (!RunningOnEmulator)
+            {
+                Task.Run(() => CreateDatabaseAsync(databaseId)).Wait();
+            }
+            FirestoreDb = new FirestoreDbBuilder
+            {
+                ProjectId = ProjectId,
+                DatabaseId = databaseId,
+                EmulatorDetection = EmulatorDetection.EmulatorOrProduction
+            }.Build();
+            CollectionPrefix = RunningOnEmulator ? IdGenerator.FromGuid(prefix: "test-") : "";
             NonQueryCollection = FirestoreDb.Collection(CollectionPrefix + "-non-query");
             HighScoreCollection = FirestoreDb.Collection(CollectionPrefix + "-high-scores");
             StudentCollection = FirestoreDb.Collection(CollectionPrefix + "-students");
```

Comment:
I think you may want to remove the `-` from the beginning of the suffixes, so -non-query => non-query, etc.

Reply:
Done, thanks.

Reply:
(Added a suffix of "-" to CollectionPrefix, too.)